app/asg.tf

```diff
@@ -0,0 +1,28 @@
+resource "aws_autoscaling_group" "wp" {
+    name                 = "${var.service}-wp-${var.environment}-asg"
+    launch_configuration = aws_launch_configuration.wp_launch_config.name
+
+    vpc_zone_identifier = [
+        var.private_subnet_ids
```

Comment:
I'm not sure if you tested it against the new output resulting from the for_each approah
private_subnet_ids = {
  "private-subnet-2a-dev" = "subnet-0e1f5746df3396cca"
  "private-subnet-2b-dev" = "subnet-0a39fef05b0d5278b"
}
It's now a map rather than a list.

Reply:
done